### PR TITLE
TA-1275: Test implementation

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**

### CODE REVIEW

File: example_suites/ta1275_acl_add_remove_tag_from_column.json [hunk 1]
```diff
@@ -0,0 +1,638 @@
...
+            "Defaults": "aclprofile.defaults",
+            "Params": [
+              {
+                "Key": "name",
+                "Value": "\"acl_with_one_tag_to_each_column\""
+              },
+              {
+                "Key": "id",
+                "Value": "\"acl_with_one_tag_to_each_column\""
+              },
```
Comment (olgasobolevsky) [inline example_suites/ta1275_acl_add_remove_tag_from_column.json:29]: For consistency, let's not use "_" and "-" in same tag/name/id.
For example:
Actual: \"allow_bot-tag\"
Expected: \"allow_bot_tag\"

File: example_suites/ta1275_acl_add_remove_tag_from_column.json [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+              {
+                "Key": "ok",
+                "Value": true
+              }
+            ]
+          }
+        },
+        {
+          "Name": "Tag with 1/2 Chars",
+          "Skip": false,
```
Comment (olgasobolevsky) [inline example_suites/ta1275_acl_add_remove_tag_from_column.json:300]: It's 2 tests:
1. For tag with 1 char
2. For tag with 2 chars

File: example_suites/ta1275_acl_add_remove_tag_from_column.json [hunk 3]
```diff
@@ -0,0 +1,638 @@
+{
+  "Name": "ACL - Add/Remove tag from column TA-1275",
+  "Skip": false,
+  "Tests": [
+    {
```
Comment (olgasobolevsky) [inline example_suites/ta1275_acl_add_remove_tag_from_column.json:5]: Please divide this 1 big Test to 6 separate Tests: one Test in pr for 1 step in TP.

File: example_suites/acl_add_remove_tag_from_column.json [hunk 4]
```diff
@@ -0,0 +1,754 @@
...
+                "Value": "\"acl_same_tag_diff_columns\""
+              },
+              {
+                "Key": "allow",
+                "Value": "[\"same-tag\"]"
+              },
+              {
+                "Key": "allow_bot",
+                "Value": "[\"same-bot-tag\"]"
+              }
```
Comment (olgasobolevsky) [inline example_suites/acl_add_remove_tag_from_column.json:632]: For this test the tag should be the same in both columns: "same-tag"

File: example_suites/acl_add_remove_tag_from_column.json [hunk 5]
```diff
@@ -0,0 +1,754 @@
+{
+  "Name": "ACL - Add/Remove tag from column TA-1275",
+  "Skip": false,
+  "Tests": [
```
Comment (olgasobolevsky) [inline example_suites/acl_add_remove_tag_from_column.json:4]: This suite should be rearranged: one test for each step in test plane (jira task page), i.e. 6 Tests.
(Not to go lost in Tests/Steps, it helps to add in the name for test: "Test: ..." and in the name for step: "Step:...")

Comment (EvgeniiNikonets) [reply example_suites/acl_add_remove_tag_from_column.json:4]: done

**Review by olgasobolevsky — APPROVED** [review]

**Pull request merged**